docker/DockerFile

```diff
@@ -0,0 +1,30 @@
+FROM ruby:2.5.5-alpine3.10
+ENV BUILD_DEPS="curl tar wget linux-headers imagemagick fontconfig bash libstdc++ libx11 libxrender libxext libssl1.1 ca-certificates fontconfig freetype ttf-dejavu ttf-droid ttf-freefont ttf-liberation ttf-ubuntu-font-family" \
```

Comment:
Are there any entries in this list that are not needed for this app? Or is that even a thing you worry about in Docker?

Reply:
@danjohnson3141 , this list looks pretty lean to me. Probably none of these have any huge dependencies. I like to refresh myself on this Docker article once in a while: https://www.ianlewis.org/en/creating-smaller-docker-images

Reply:
Getting the band back together :) 🎹

Reply:
Whoa, a Caputo guest appearance!

Reply:
:D When I saw that this was on a repo that I could see, I couldn't help but take a peek.